test/modules/core/shaderlib/shadow/shadow.spec.js

```diff
@@ -5,6 +5,67 @@ import {shadow} from '@deck.gl/core/shaderlib';
 import {Matrix4, Vector3} from 'math.gl';
 import {PROJECT_COORDINATE_SYSTEM} from '@deck.gl/core/shaderlib/project/constants';
 
+const TEST_CASE1 = [
+  {
+    xyz: [83095, 202499, 0],
```

Comment:
Use lnglat coordinates and test with multiple viewports (z <12 and z > 12)

Reply:
it is done, TEST_CASE1 is for zoom == 10, TEST_CASE2 is for zoom == 15

Reply:
You need to document how these numbers are calculated.

Reply:
add some comments for test cases